R/db.R

```diff
@@ -107,9 +107,14 @@ setMethod("sqlCreateTable", "Teradata",
 
 setMethod(
   "dbListTables", "Teradata",
-  function(conn, ...) {
+  function(conn, catalog_name = NULL, schema_name = NULL, table_name = NULL,
+    table_type = NULL, ...) {
     c(dbGetQuery(conn, "HELP VOLATILE TABLE")[["Table SQL Name"]],
-      odbcConnectionTables(conn, ...)$table_name)
+      odbcConnectionTables(conn,
```

Comment:
Might be worth pulling this out into a variable since it's a bit hard to parse currently.

Reply:
Agreed.

Sounds like we may need to circle back to revisit some of the other S4/Teradata extensions.  I'll revisit the syntax here then.